### PR TITLE
utils: Add function to check for a config file

### LINKED ISSUE #1242: GUI settings watch fails if no client settings were made yet
```
ricab@ricab-xps13:~ $ snap install --classic --channel beta/1.0 multipass
multipass (beta/1.0) 1.0.0 from Canonical✓ installed
ricab@ricab-xps13:~ $ multipass.gui
QStandardPaths: XDG_RUNTIME_DIR points to non-existing path '/run/user/1000/snap.multipass', please create it with 0700 permissions.
[2019-12-18T13:02:39.797] [warning] [Qt] inotify_add_watch("/home/ricab/snap/multipass/1525/config/multipass/multipass.conf") failed: "No such file or directory"
```

The GUI then does not see subsequent `multipass set client.gui.autostart=false` until it is restarted.

### PR DESCRIPTION
**Pull request by townsend2010**
Creates an empty config file if it doesn't exists.

Fixes #1242

### CODE REVIEW

File: tests/test_utils.cpp [hunk 1]
```diff
@@ -370,3 +370,13 @@ TEST(Utils, vm_stopped_returns_false)
 
     EXPECT_FALSE(mp::utils::is_running(state));
 }
+
+TEST(Utils, no_config_file_and_dir_are_created)
```
Comment (ricab) [inline tests/test_utils.cpp]:
```suggestion
TEST(Utils, absent_config_file_and_dir_are_created)
```

**Comment by townsend2010** [comment]
@ricab, Sure, I'll do the `lastModifed` test and change the test name.

**Comment by ricab** [comment]
@townsend2010 thanks! Just tested the snap and all works well :)

**Comment by townsend2010** [comment]
@ricab, both parts are done.  Thanks!

File: tests/test_utils.cpp (hunk 1 above)
Comment (townsend2010) [reply tests/test_utils.cpp]: Done

**Comment by ricab** [comment]
Unfortunately bors won't like this because of the merge conflict in full...

**Comment by ricab** [comment]
I was reminded bors is actually out after the org move, so merging will have to be done manually in 2020. The mac build has been restarted and we should get a mac package soon.

**Comment by Saviq** [comment]
Okies,

bors r+

**Pull request merged**